tapir/shifts/views/management.py

```diff
@@ -239,10 +238,16 @@ def get_context_data(self, **kwargs):
         )
         context = super().get_context_data(**kwargs)
         context["card_title"] = _("Duplicate ABCD-Shift " + template.get_display_name())
-        context["help_text"] = _(
+        help_text = _(
             "Please choose other weekdays and weekgroups this ABCD-Shift should be copied to. These slots will be copied: "
```

Comment:
It's small but the text says "other weekdays and weekgroups",  but the user is able to select the same weekgroup and weekday :laughing: